Service/ResponseTransformerStrategy.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Auto1\ServiceAPIClientBundle\Service;
+
+use Auto1\ServiceAPIComponentsBundle\Exception\AbstractException;
+use Auto1\ServiceAPIComponentsBundle\Service\Endpoint\EndpointInterface;
+use Psr\Http\Message\ResponseInterface;
+
+interface ResponseTransformerStrategy
```

Comment:
lets keep the consistency and add the Interface prefix

Reply:
renamed